src/models/callStack.js

```diff
@@ -20,21 +20,64 @@ module.exports = class CallStack {
 
   // Attempts to remove any sensitive data that may be found within
   sanitize(content) {
-    if (typeof content !== "object") {
-      return content;
-    }
 
+    const hideString = "*****";
     let outContent = {};
+    let type = typeof content;
+
+    // Since JavaScript `typeof` will assign an array as "object" as well as null
+    // we will extend this typeof check to add those as different types, to ease
+    // the complexity of the below switch statement
+    if (Array.isArray(content)) {
+      type = "array";
+    }
+    if (content === null) {
```

Comment:
another microoptimisation: if its an array it won't be null, so this could be `else if`

Reply:
Also fantastic idea, like you said micro-optimization, but everything helps